experiments/src/main/scala/HelloZio/CalculatorExample.scala

```diff
@@ -114,15 +114,16 @@ object CalculatorExample extends zio.App:
     String
   ] =
     for
-      (number1, number2) <-
-        ZIO {
+      number <- // note that `(number1, number2)` now results in: value withFilter is not a member of zio.ZIO[Any, Nothing, (Float, Float)], but could be made available as an extension method.
```

Comment:
Yep, they talked about how removing `withFilter` making the ZIO types more uniform.

Reply:
Not sure why destructuring the tuple requires the `withFilter`. Could be a Scala bug or unfortunate need.

Reply:
I've seen chatter about it before on the Scala mailing lists, and it seems to be an unfortunate need.